### ISSUE TITLE
django.core.exceptions.ImproperlyConfigured after running python manage.py collectstatic

### ISSUE THREAD

**Issue by Kasramvd** [ASKBOT/askbot-devel#562]
after following all the documentation and running the following command:

    python manage.py collectstatic

I keep getting the following Error:

```
WARNING!!! You are using a 'locmem' (local memory) caching backend,
which is OK for a low volume site running on a single-process server.
For a multi-process configuration it is neccessary to have a production
cache system, such as redis or memcached.

With local memory caching and multi-process setup you might intermittently
see outdated content on your site.

Please run command

    python manage.py collectstatic
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/Django-1.8.9-py2.7.egg/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/Django-1.8.9-py2.7.egg/django/core/management/__init__.py", line 328, in execute
    django.setup()
  File "/usr/local/lib/python2.7/dist-packages/Django-1.8.9-py2.7.egg/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/dist-packages/Django-1.8.9-py2.7.egg/django/apps/registry.py", line 115, in populate
    app_config.ready()
  File "/usr/local/lib/python2.7/dist-packages/Django-1.8.9-py2.7.egg/django/contrib/admin/apps.py", line 22, in ready
    self.module.autodiscover()
  File "/usr/local/lib/python2.7/dist-packages/Django-1.8.9-py2.7.egg/django/contrib/admin/__init__.py", line 24, in autodiscover
    autodiscover_modules('admin', register_to=site)
  File "/usr/local/lib/python2.7/dist-packages/Django-1.8.9-py2.7.egg/django/utils/module_loading.py", line 74, in autodiscover_modules
    import_module('%s.%s' % (app_config.name, module_to_search))
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/local/lib/python2.7/dist-packages/djcelery/admin.py", line 259, in <module>
    class PeriodicTaskAdmin(admin.ModelAdmin):
  File "/usr/local/lib/python2.7/dist-packages/djcelery/admin.py", line 261, in PeriodicTaskAdmin
    form = periodic_task_form()
  File "/usr/local/lib/python2.7/dist-packages/djcelery/admin.py", line 236, in periodic_task_form
    class PeriodicTaskForm(forms.ModelForm):
  File "/usr/local/lib/python2.7/dist-packages/Django-1.8.9-py2.7.egg/django/forms/models.py", line 274, in __new__
    "needs updating." % name
django.core.exceptions.ImproperlyConfigured: Creating a ModelForm without either the 'fields' attribute or the 'exclude' attribute is prohibited; form PeriodicTaskForm needs updating.
```

Do you know that what's the underlying reason?

**Comment by evgenyfadeev** [comment]
Please try the latest code from the master branch, fixed version of django-celery there (3.1.17)
Could you please close the issue if this helps?

**Comment by Kasramvd** [comment]
@evgenyfadeev  Thank you so much.  It worked like a charm :-)
